framework/db/Connection.php

```diff
@@ -618,9 +624,10 @@ protected function initConnection()
      */
     public function createCommand($sql = null, $params = [])
     {
-        $command = new Command([
+        $command = Yii::createObject([
```

Comment:
I really think the consistency is worth more than this minor and mostly theoretical performance gain.
It is literally a few function calls and isset calls.